lib/main_screen.dart

```diff
@@ -29,109 +29,128 @@ class MainScreenState extends State<MainScreen> {
   @override
   Widget build(BuildContext context) {
     GlobalBloc globalBloc = GlobalProvider.of(context);
+    double width = MediaQuery.of(context).size.width;
     return Scaffold(
-      appBar: AppBar(
-        leading: Image.asset(
-          'assets/eventhub-logo2.png',
-          scale: 8,
-        ),
-        title: Text("EventHub"),
-        actions: <Widget>[
-          IconButton(
-            icon: Icon(Icons.info_outline),
-            onPressed: () {
-              showAboutDialog(
-                  context: context,
-                  applicationVersion: "1.0",
-                  applicationIcon: Card(
-                      color: Colors.red[800],
-                      child: Padding(
-                        padding: const EdgeInsets.all(8.0),
-                        child: Image.asset(
-                          'assets/eventhub-logo2.png',
-                          scale: 8,
-                        ),
-                      )),
-                  applicationName: 'EventHub',
-                  children: [
-                    Linkify(
-                      text: '''
+      backgroundColor: Color(0xffB54646),
+      body: Center(
+        child: Container(
+          decoration: BoxDecoration(
+            color: Colors.transparent,
```

Comment:
Instead of repeating the same thing again and again, it's best to make a new custom widget (maybe something called `SizeController`, you can decide the name) and wrapping every screen with this widget.

Reply:
Also in this custom widget, only do this if the variable `kIsWeb` is true, otherwise, just pass the underlying child as it is for mobile apps.

Reply:
I didn't find any need to make a new widget. It will only increse number of file for no reason.

Reply:
Increasing number of files doesn't matter.

Creating a new widget is helpful because if we want to change the width later on, we can simply make the change in 1 single place rather than having to make it everywhere.